en/streaming-integrator/docs/develop/installing-siddhi-extensions.md

```diff
@@ -273,85 +200,53 @@ This is an _array_. Each member of this _array_ is an _object_, which denotes in
       <code>name</code>
     </td>
     <td>
-      Uniquely identifiable name of the dependency. If this dependency denotes the jar of the Siddhi extension itself, this starts with <code>siddhi-</code>.
+      The uniquely identifiable name of the dependency. If this dependency denotes the jar of the Siddhi extension itself, it starts with <code>siddhi-</code>.
     </td>
   </tr>
   <tr>
     <td>
       <code>version</code>
     </td>
     <td>
-      Version of the dependency.
+      The version of the dependency.
     </td>
   </tr>
   <tr>
     <td><code>download</code></td>
     <td>
-      Denotes download information of the dependency, through the following properties.
-      <table>
-        <tr>
-          <td>
-            <code>autoDownloadable</code>
-          </td>
-          <td>
-            Whether the dependency is auto downloadable (<code>true</code>) or not (<code>false</code> - <a href="#manually-installable-dependencies">manually installable</a>).
-          </td>
-        </tr>
-        <tr>
-          <td>
-            <code>url</code>
-          </td>
-          <td>
-            <b>Applicable only if the dependency is auto downloadable.</b> The download URL to download the dependency's jar.
-          </td>
-        </tr>
-        <tr>
-          <td>
-            <code>instructions</code>
-          </td>
-          <td>
-            <b>Applicable only if the dependency is <a href="#manually-installable-dependencies">manually installable</a>.</b> Instructions to download (and convert) the dependency's jar.
-          </td>
-        </tr>
-      </table>
+      <p>This denotes download information of the dependency via the following properties.</p>
+      <ul>
+        <li><code>autoDownloadable</code>: This specifies whether the dependency is auto downloadable via the <code>true</code> and <code>false</code> values. If the value is <code>false</code>, the property is <a href="#manually-installable-dependencies">manually installable</a>.</li>
+        <li><code>url</code>: <strong>If the dependency is auto downloadable</strong>, this specifies the URL via which you can download the JAR of the dependency.</li>
```

Comment:
Shall we rephrase _"this specifies the URL via which you can download the JAR of the dependency"_, since this URL is used to download the jar automatically, not manually (_"... you can download ..."_ seems misleading).

Reply:
Done